styles/DataCleanupPage.module.scss

```diff
@@ -322,3 +322,36 @@ $long-field-multiplyer: 1.75;
   height: auto;
   max-height: ($field-height * 3) + 1rem;
 }
+
+.phoneContainer {
+  margin: 0.2rem !important;
+}
+
+.phoneInput {
+  height: $field-height !important;
```

Comment:
Why all these `!important`s? Best practice is to use these really sparingly so having entire classes full of them is suspect.

Also when you use variables like these in SCSS run `yarn build` after, last time I built I had a few issues with Variables in the SCSS